README.md

```diff
@@ -64,6 +64,16 @@ gyms - process gyms in GMO
 pokestops - process pokestops in GMO
 cells - process cell updates (disabling this also disables automatic fort clearance)
 
+# PvP
+Extra configurations for PvP are available in the `pvp` section of the config file.
+
+```toml
+# The level caps used in rankings, defaults to [50, 51]
+level_caps = [50, 51]
+# Whether little cup rules should be applied to the 500cp league, default to true
+use_little_cup_rules = true
```

Comment:
This means now we allow evolutions or not? What's the little cup rules in general? :)